build.gradle

```diff
@@ -542,6 +541,7 @@ def sonarExclusions = [
   '**uk/gov/hmcts/reform/prl/services/**/*',
   '**uk/gov/hmcts/reform/prl/controllers/citizen/*',
   '**uk/gov/hmcts/reform/prl/services/citizen/*',
+  '**uk/gov/hmcts/reform/prl/services/ManageOrderService.java',
   '**uk/gov/hmcts/reform/prl/repositories/**/*'
```

Comment:
we are trying to address this in 604

Reply:
You can rebased with master once this PR is merged please.
If we remove these our build will fail & we won't be able to merge PR